Source/WebCore/page/DebugPageOverlays.cpp

```diff
@@ -433,7 +433,7 @@ void InteractionRegionOverlay::drawRect(PageOverlay&, GraphicsContext& context,
         context.setStrokeColor(Color::green);
 
         for (const auto& region : m_regions) {
-            for (const auto& rect : region.rectsInContentCoordinates)
+            for (const auto& rect : region.regionInLayerCoordinates.rects())
             context.strokeRect(rect, 2);
```

Comment:
Nit - not new, but the indentation level is off here.

Reply:
Weird! Fixing.